pkg/codegen/testing/test/program_driver.go

```diff
@@ -200,6 +204,85 @@ var PulumiPulumiProgramTests = []ProgramTest{
 		Skip:        codegen.NewStringSet("go"),
 		// Blocked on go: TODO[pulumi/pulumi#10834]
 	},
+	// PCL files from pulumi/yaml transpiled examples
+	{
+		Directory:   "transpiled_examples/aws-eks",
```

Comment:
See an above comment for the definition of `transpiled`.
```suggestion
		Directory:   transpiled("aws-eks"),
```